.github/workflows/main.yml

```
@@ -11,11 +11,12 @@ permissions:
 
 jobs:
   test:
-    name: Test - Python ${{ matrix.python-version }}
-    runs-on: ubuntu-latest
+    name: Test - Python ${{ matrix.python-version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
```

It can be _slightly_ quicker to start the slower ones first:
```suggestion
        os: ["windows-latest", "macos-latest", "ubuntu-latest"]
```